ssc/cmod_utilityrate5.cpp

```diff
@@ -1787,6 +1786,15 @@ class cm_utilityrate5 : public compute_module
 			ssc_number_t dollars_applied = 0;
 			if (enable_nm)
 			{
+                if (rollover_credit && m == net_metering_credit_month)
+                {
+                    ssc_number_t year_end_dollars = monthly_cumulative_excess_energy[m] * as_number("ur_nm_yearend_sell_rate") * rate_esc;
+                    payment[c - 1] -= year_end_dollars;
+                    monthly_ec_charges[m] -= year_end_dollars;
+                    excess_dollars_earned[m] += year_end_dollars;
+                    dollars_applied += year_end_dollars;
```

Comment:
Are the monthly, hourly, and bill values matching?

Reply:
Yes.